### PR TITLE
Fix image sources in Tines readme

### PR DESCRIPTION
**Pull request by bmorelli25**
The Tines integration screenshot sources are incorrect. This PR fixes them so the screenshots will correctly render in Kibana and our documentation.

### CODE REVIEW

File: packages/tines/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.0.2"
+  changes:
+    - description: Fix img link in readme
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/5076
```
Comment (bmorelli25) [inline packages/tines/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/5415
```

File: packages/tines/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.0.2"
+  changes:
+    - description: Fix img link in readme
+      type: enhancement
```
Comment (bhapas) [inline packages/tines/changelog.yml:5]: A bugfix?

Comment (bmorelli25) [reply packages/tines/changelog.yml:5]:
```suggestion
      type: bugfix
```

**Review by bhapas — APPROVED** [review]
LGTM

**Pull request merged**